.github/workflows/deploy.yml

```diff
@@ -61,6 +61,7 @@ jobs:
         RENKU_BOT_DEV_PASSWORD: ${{ secrets.RENKU_BOT_DEV_PASSWORD }}
       run: |
         cd tests
+        COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose build sbt
```

Comment:
I don't know why, but setting the same environment variable on the `docker-compose run` command directly triggers a build using the standard build kit, therefore the two line.

Reply:
you don't have to specify which image to use for the cache?

Reply:
oh nevermind, it just uses `latest` by default